generic-sensor/idlharness.https.window.js

```diff
@@ -0,0 +1,29 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+// https://w3c.github.io/sensors/
+
+'use strict';
+
+function cast(i, t) {
+  return Object.assign(Object.create(t.prototype), i);
+}
+
+promise_test(async () => {
```

Comment:
@lukebjerring, thanks for the improvement!
Can we use `idl_test` helper, it's more simplified. :)

Reply:
Done.